lib/web/apiserver.go

```diff
@@ -2216,8 +2217,57 @@ func (h *Handler) siteNodeConnect(
 		return nil, trace.Wrap(err)
 	}
 
-	h.log.Debugf("New terminal request for ns=%s, server=%s, login=%s, sid=%s, websid=%s.",
-		req.Namespace, req.Server, req.Login, req.SessionID, ctx.GetSessionID())
+	// Fetching nodes is quite slow when many nodes exist so fetch it once and
```

Comment:
Is this comment misplaced? It talks about fetching nodes, but it's sitting next to code that generates a client.

Reply:
Yep, removed.